client-side/src/components/Report/report.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PdfGenerator from './pdf.jsx';
+import { useState } from 'react';
```

Comment:
please collect all the react imports to one that will be the first import in the page